src/diffusers/models/modeling_utils.py

```diff
@@ -761,6 +763,42 @@ def _find_mismatched_keys(
 
         return model, missing_keys, unexpected_keys, mismatched_keys, error_msgs
 
+    @classmethod
+    def convert_deprecated_attention_blocks(cls, state_dict):
```

Comment:
After we load the state dict, we can just manually rename the layers ({query, key, value, out projection) to the new layer names.